prow/config.yaml

```diff
@@ -5634,6 +5634,30 @@ postsubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+    - name: ci-kubernetes-e2e-kubeadm-gce-1-9
+      agent: kubernetes
+      labels:
+        preset-service-account: true
+        preset-k8s-ssh: true
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180226-208823aed
+          args:
+          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+          - "--clean"
+          - "--git-cache=/root/.cache/git"
```

Comment:
drop this flag and the volume / mount / host port, we're trying to eliminate this and normally most CI tests don't use this

Reply:
*please* :-), I'll cleanup some more of the other jobs in a follow-up